test/config/updatedeletetest.go

```diff
@@ -36,48 +31,40 @@ const (
 
 // TestUpdateDelete tests update and delete paths in a single GNMI request
 func (s *TestSuite) TestUpdateDelete(t *testing.T) {
-	// Get the first configured target from the environment.
-	target := gnmiutils.CreateSimulator(t)
-	defer gnmiutils.DeleteSimulator(t, target)
+	// Get the first configured simulator from the environment.
+	simulator := gnmiutils.CreateSimulator(t)
+	defer gnmiutils.DeleteSimulator(t, simulator)
 
 	// Make a GNMI client to use for requests
 	gnmiClient := gnmiutils.GetGNMIClientOrFail(t)
 
 	// Create interface tree using gNMI client
 	setNamePath := []proto.TargetPath{
-		{TargetName: target.Name(), Path: udtestNamePath, PathDataValue: udtestNameValue, PathDataType: proto.StringVal},
+		{TargetName: simulator.Name(), Path: udtestNamePath, PathDataValue: udtestNameValue, PathDataType: proto.StringVal},
 	}
-	gnmiutils.SetGNMIValueOrFail(t, gnmiClient, setNamePath, gnmiutils.NoPaths, gnmiutils.NoExtensions)
-
-	err := gnmiutils.WaitForConfigurationCompleteOrFail(t, configapi.ConfigurationID(target.Name()), time.Minute)
-	assert.NoError(t, err)
+	gnmiutils.SetGNMIValueOrFail(t, gnmiClient, setNamePath, gnmiutils.NoPaths, gnmiutils.SyncExtension(t))
```

Comment:
If gnmiutils.WaitForConfigurationCompleteOrFail is still under util we can remove the function itself to make sure there is no call to it.

Reply:
We still need this for offline target tests.